src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,13 +1,30 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
+    private double bladeWidth;
+    private double weight;
+
     public Bulldozer() {
     }
 
```

Comment:
You should add this kind of constructor to ALL machines that `extends Machine` (Example below)

![image](https://github.com/user-attachments/assets/74a9e46a-ef85-4e67-afcc-07a98fcdef51)